src/scripts/Game.ts

```diff
@@ -100,6 +100,8 @@ class Game {
 
         // If the player isn't on a route, they're in a town/dungeon
         this.gameState = player.route() ? GameConstants.GameState.fighting : GameConstants.GameState.town;
+
+        document.dispatchEvent(new CustomEvent('pokeclicker.game.initialized'));
```

Comment:
As per below, use `SpecialEvents.initialize()` to be more consistent with other `Feature`.
```suggestion
```